packages/client/hmi-client/src/services/project.ts

```diff
@@ -90,6 +92,9 @@ async function getAssets(projectId: string, types?: string[]): Promise<ProjectAs
 				// add URL with format: ...?types=A&types=B&types=C
 				url += `${indx === 0 ? '?' : '&'}types=${type}`;
 			});
+		} else {
+			url +=
+				'?types=datasets&types=model_configurations&types=models&types=publications&types=simulations&types=workflows';
```

Comment:
Can we to opt-out semantics? If none is specified assume we want to retrieve everything? This makes it easier to add additional asset types.

Reply:
I believe that TDS's old API took no types as "return everything"... where as now it returns an error and requires an explicit type(s).  There is absolutely better ways to do this that I want to circle back to - right now i just need to get this working :)